### DISCUSSION TITLE
Add support for Dadamachines Doppler Board (5k)

### DISCUSSION THREAD

**Discussion by DatanoiseTV** [FPGAwars/icestudio#822] [New Boards Requests]
It would be fantastic to have bitstream export support with the right pinmapping for doppler.
https://github.com/dadamachines/doppler/blob/master/hardware/doppler-quickstart_print.pdf
https://github.com/dadamachines/doppler/blob/master/hardware/dadamachnes-doppler-schematic.pdf

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
